lalrpop/src/build/action.rs

```diff
@@ -359,12 +364,17 @@ fn emit_inline_action_code<W: Write>(
                 // execute the inlined reduce action
                 rust!(
                     rust,
-                    "let {}temp{} = {}action{}(",
+                    "let {}temp{} = {}action{}{}",
                     grammar.prefix,
                     temp_counter,
                     grammar.prefix,
-                    inlined_action.index()
+                    inlined_action.index(),
+                    lparen
                 );
+                for t in grammar.non_lifetime_type_parameters() {
+                    rust!(rust, "{},", t);
+                }
+                if annotate {rust!(rust, ">(")};
```

Comment:
Might be a bit cleaner to go

```rust
if annotate {
    rust!(rust, "(");
} else {
    rust!(rust, "::<")};
    for t in grammar.non_lifetime_type_parameters() {
        rust!(rust, "{},", t);
    }
    rust!(rust, ">(");
}
```

Reply:
This also breaks the CI, since the parenthesis is added on a new line now. Which, as I think about it is exactly the reason that I went with the weird conditional formatting. I can just be OK with changing the default output code, since it is still the same semantically. Should I regenerate the snapshot?

Reply:
Of course if I'm going to regenerate the snapshot then it maybe makes sense to unconditionally add the type annotation and just have it be empty most of the time.

Reply:
Ah, didn't realize it would put it on a separate line. Just reset to the original code in that case.